ugs-pendant/src/main/java/com/willwinder/universalgcodesender/pendantui/v1/controllers/SettingsController.java

```diff
@@ -0,0 +1,53 @@
+package com.willwinder.universalgcodesender.pendantui.v1.controllers;
+
+import com.willwinder.universalgcodesender.model.BackendAPI;
+import com.willwinder.universalgcodesender.pendantui.v1.model.Settings;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Path("/v1/settings")
+public class SettingsController {
+
+    @Inject
+    private BackendAPI backendAPI;
+
+    @GET
+    @Path("getSettings")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getSettings() {
+        try {
+            com.willwinder.universalgcodesender.utils.Settings settings = backendAPI.getSettings();
+            Settings response = new Settings();
+            response.setJogFeedRate(settings.getJogFeedRate());
+            response.setJogStepSizeXY(settings.getManualModeStepSize());
+            response.setJogStepSizeZ(settings.getzJogStepSize());
+            response.setPreferredUnits(settings.getPreferredUnits());
+            return Response.ok(response).build();
```

Comment:
Can you just return "Settings" here? I'm not sure exactly what magic Dropwizard adds, but it has always let me return POJOs directly rather than wrap them in a Response.

Reply:
I also find that makes the code easier to read when the signature tells you the return type (not a big deal for such a small function)